packages/insomnia-app/app/plugins/install.js

```diff
@@ -170,6 +172,37 @@ async function _installPluginToTmpDir(lookupName: string): Promise<{ tmpDir: str
   });
 }
 
+function isFalsePositive(stderr) {
+  // Split on line breaks and remove falsy values (null, undefined, 0, -0, NaN, "", false)
+  const arr = stderr.split(/\r?\n/).filter(e => e);
+  // Retrieve all matching deprecated dependency warning
+  const warnings = arr.filter(e => isDeprecatedDependencies(e));
+  // Print each deprecation warnings to the console, so we don't hide them.
+  warnings.forEach(e => console.log(e));
```

Comment:
We tend to scope console messages, so I propose the following change. Could you please add a screenshot for messages in the console after you try to install your plugin?

```suggestion
  warnings.forEach(e => console.log('[plugins] deprecation warning during installation: ', e));
```

Reply:
![image](https://user-images.githubusercontent.com/20780192/100400092-88c69080-3055-11eb-9d99-711d8714dfc5.png)
**Here is it**

Reply:
Thanks! We should probably change that to `console.warn` instead of `console.log` so it applies colors accordingly. But looks good!